components/annot-columns.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <b-card no-body>
+    <!--    TODO: make this also toggleable like the explanation tab-->
+    <b-card-header>Review the annotated columns</b-card-header>
+    <b-card-body style="height:30vh; overflow-y:scroll;">
+      <b-list-group>
+        <b-list-group-item
+          class="d-flex justify-content-between align-items-center"
+          v-for="[columName, categoryName] of Object.entries(filteredColumns)">
```

Comment:
I'm getting the following warning from here:

>  WARN  in ./components/annot-columns.vue?vue&type=template&id=6b92ec6b&        friendly-errors 10:51:55
> 
> Module Warning (from ./node_modules/vue-loader/lib/loaders/templateLoader.js): friendly-errors 10:51:55
> (Emitted value instead of an instance of Error) <b-list-group-item v-for="[columName, categoryName] in Object.entries(filteredColumns)">: component lists rendered with v-for should have explicit keys. See https://vuejs.org/guide/list.html#key for more info.